toolchain/lower/testdata/interop/cpp/function_decl.carbon

```diff
@@ -22,6 +26,25 @@ fn MyF() {
   Cpp.foo();
 }
 
+// ============================================================================
+// inline_function_decl
+// ============================================================================
+
+// --- inline_function_decl.h
+
+inline void foo() {}
+
+// --- todo_import_inline_function_decl.carbon
+
+library "[[@TEST_NAME]]";
+
+import Cpp library "inline_function_decl.h";
+
+fn MyF() {
+  // TODO: This should generate the definition of the inline function `foo()`.
```

Comment:
Should this move to the new function_decl_inline.carbon copy of this test? I'm having trouble seeing a difference other than this TODO, are they duplicates now?

Reply:
This file is under the `lower` directory, while `function_decl_inline` is under the `check` directory, so they test different parts, while the input is similar.

Reply:
Ohh thanks. okay. I'll let Jon finish his review, but LGTM

Reply:
Thanks!